x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/index.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { memo, useState, useMemo } from 'react';
+import React from 'react';
+import { EuiDataGrid } from '@elastic/eui';
+import { useDispatch, useSelector } from 'react-redux';
+import { AlertAction } from '../../store/alerts/action';
+import * as selectors from '../../store/selectors';
+import { usePageId } from '../use_page_id';
+
+export const AlertIndex = memo(() => {
+  usePageId('alertsPage');
```

Comment:
This is where we call the `usePageId` hook, which will dispatch a `userNavigatedToPage` action.